src/StreamJsonRpc/ActivityTracingStrategy.cs

```diff
@@ -40,6 +59,22 @@ public void ApplyOutboundActivity(JsonRpcRequest request)
         return state;
     }
 
+    private Activity CreateNewActivity(string name)
+    {
+        Activity? activity = null;
+        if (this.activitySource is object)
+        {
+            activity = this.activitySource.StartActivity(name);
+        }
+
+        if (activity is null)
+        {
+            activity = new Activity(name);
+        }
+
+        return activity;
+    }
```

Comment:
nit: this can all be one line.

```suggestion
    private Activity CreateNewActivity(string name) => this.activitySource?.StartActivity(name) ?? new Activity(name);
```